### PR TITLE
Feature/restaurant page

### PR DESCRIPTION
**Pull request by jpdf00**
The page is dynamically rendered and the content switches from a Home Page, a Menu Page, and a Contact Page.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/jpdf00/restaurant-page/issues"
+  },
+  "homepage": "https://github.com/jpdf00/restaurant-page#readme",
+  "devDependencies": {
+    "webpack": "^5.33.2",
+    "webpack-cli": "^4.6.0"
```
Comment (addod19) [inline package.json:25]: Kindly add the `webpack-dev-server` so we can enable hot reload functionality

File: package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (addod19) [inline package.json:10]: Kindly add these scripts `"dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
`  so we can run commands like `npm run dev/npm run build`

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @jpdf00, congratulations on this project :muscle: All the modules have been well implemented using the `createElement` and adding appropriate styling to the modules :+1: 
The live link works as expected and the commit messages are also easy to understand. :slightly_smiling_face: However, there are a few comments we need to quick fix. Kindly check the comments below:
- Kindly add the "webpack-dev-server" to enable live reload
- Also, kindly add the necessary script so we can run commands like "npm run dev" etc

Once these are addressed, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by mahmoud717 — APPROVED** [review]
### STATUS: APPROVED ✔️ ✔️ 

**hi @jpdf00**

**Great job with making all of the requirements. the project is approved, you can go ahead and merge this pull request**

**Pull request merged**